spec/controllers/admin/categories_controller_spec.rb

```diff
@@ -16,6 +17,47 @@
     assert_response :redirect, :action => 'index'
   end
 
+  describe "test_new" do
+    before(:each) do
+      get :new, :format => 'html'
+    end
+
+    it 'should render template new' do
+      assert_template 'new'
+      assert_tag :tag => "table",
+        :attributes => { :id => "category_container" }
+    end
+
+    it 'should have a new category' do
+      assigns(:category).should_not be_nil
+      assert assigns(:category).id.should be_nil
```

Comment:
assigns(:category) is referring to the instance variable @category set by the controller method.